src/main/java/org/broadinstitute/dsde/consent/ontology/resources/DataUseResource.java

```diff
@@ -34,4 +40,17 @@ public Response getSchemaV3() {
         return Response.ok().entity(content).type(MediaType.APPLICATION_JSON).build();
     }
 
+    @POST
+    @Path("/data-use/v3")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response validateSchemaV3(String json) {
+      List<String> errors = jsonSchemaUtil.validateDataUseV3Schema(json);
+      if (errors.isEmpty()) {
+        return Response.ok().type(MediaType.APPLICATION_JSON).build();
+      } else {
+        // nosemgrep
+        return Response.status(HttpStatusCodes.STATUS_CODE_BAD_REQUEST).entity(errors).type(MediaType.APPLICATION_JSON).build();
+      }
+    }
```

Comment:
We need an addition to `api-docs.yaml` so that this endpoint shows up on the swagger page.